node_modules/oae-activity/emailTemplates/mail.html.jst

```diff
@@ -33,15 +33,37 @@
      * Render the email footer
      */
     var renderFooter = function() {
-        var copyrightSymbolI18n = util.i18n.translate('__MSG__COPYRIGHT_SYMBOL__');
-        var apereoFoundationI18n = util.i18n.translate('__MSG__APEREO_FOUNDATION__');
+        var oaeNameI18n = util.i18n.translate('__MSG__OPEN_ACADEMIC_ENVIRONMENT__');
         var url = baseUrl + '/me?emailpreferences';
 
+        var oaeLink = '<a href="http://www.oaeproject.org">' + oaeNameI18n + '</a>';
+        var instanceLink = (instance.URL) ? '<a href="' + instance.URL + '">' + util.html.encodeForHTML(instance.name) + '</a>' :
```

Comment:
Didn't realize `encodeForHtml` wouldn't fail if it received `null`/`undefined`.

Note that it doesn't mimic the UI as the UI doesn't seem to encode for HTML, which is probably a bug. I'll try and verify.